README.md

```diff
@@ -224,6 +224,24 @@ Sends battery information to the server.
 | 6        | Desconnection                | 0                   |
 | 2        | Sleeping value (unused)      | 0$                  |
 
+#### BT
```

Comment:
He, he... It is probably related to other topics I'm working on but the first thing which came to my mind was: "BT???, Bluetooth???" :) Maybe some reference about what that acronym means is worth it... :) I found the reference to the "TT" one but not to the "BT" one... :)